README.md

```diff
@@ -1,10 +1,10 @@
 # Bug
 
-[!GoDoc(https://godoc.org/github.com/driusan/bug?status.svg)](https://godoc.org/github.com/driusan/bug) [![Build Status](https://travis-ci.org/driusan/bug.svg?branch=master)](https://travis-ci.org/driusan/bug) [![Test Coverage](https://codecov.io/gh/driusan/branch/master/graph/badge.svg)](https://codecov.io/gh/driusan/bug)
+[![GoDoc](https://godoc.org/github.com/driusan/bug?status.svg)](https://godoc.org/github.com/driusan/bug) [![Build Status](https://travis-ci.org/driusan/bug.svg?branch=master)](https://travis-ci.org/driusan/bug) [![Test Coverage](https://codecov.io/gh/driusan/bug/branch/master/graphs/badge.svg)](https://codecov.io/gh/driusan/bug)
 
 bug writes code problem reports to plain text files.
 
-bug requires Go version 1.7 or greater.
+bug requires Go version 1.9 or greater.
```

Comment:
When did that happen?

Reply:
The dependency go-github said it only required 1.7 so I tried it and the code worked locally. I thought it was OK and updated the README and code. I then saw that the travis-ci.org builds fail. Version 1.9 is what travis-ci says it requires and 1.9 makes the builds succeed. I updated the code and issued the previous pull request since it passed. I forgot to update this README as described in this [bug 17 comment](https://github.com/driusan/bug/pull/17/#issuecomment-440221623)

Reply:
Go-github is only required for the import tool, but I guess it doesn't hurt to document it as the minimum requirement